test/unit/application_keys_test.rb

```diff
@@ -191,6 +191,9 @@ def test_regenerate_key
     assert_in_delta expected_updated_at, @application.updated_at, 1.second
   end
 
+  # NOTE: with Rails 6.1 default has_many_inversing = true this test fails with the issue:
+  # Minitest::UnexpectedError: NoMethodError: undefined method `admins' for nil:NilClass
+  #     app/messengers/contract_messenger.rb:7:in `setup'
```

Comment:
I wonder if this message is useful here. If we change the setting without fiind the underlying issue, then the tests will fails. And whoever is doing that will fix the underlying issue.

Reply:
I don't mind removing it.

Reply:
On the other hand, when these tests started failing for me, I didn't realize immediately that this is related to `has_many_inversing` setting :shrug: 

But that's fine.

Reply:
Do you want me to remove all comments about this setting from the tests @akostadinov ?